### ISSUE TITLE
when adding modelBuilder.UseCollation("foobar") EF generates code which don't compile

### ISSUE THREAD

**Issue by vmachacek** [dotnet/efcore#23962]
When adding modelBuilder.UseCollation("foobar") EF generates migration which results in uncompilable code.

```
public partial class collation : Migration
    {
        protected override void Up(MigrationBuilder migrationBuilder)
        {
            migrationBuilder.AlterDatabase(
                collation: "sql_latin1_general_cp1_ci_as");
        }

        protected override void Down(MigrationBuilder migrationBuilder)
        {
            migrationBuilder.AlterDatabase(,
                oldCollation: "sql_latin1_general_cp1_ci_as");
        }
    }
```

` migrationBuilder.AlterDatabase(,` comma is the problem

## steps to reproduce:

1. clone this repository https://github.com/vmachacek/ef-collation-bug
2. correct connection string
3. run `dotnet ef migrations add collation`
4. project won't compile

project was built following this article
https://docs.microsoft.com/en-us/ef/core/get-started/overview/first-app?tabs=netcore-cli

**Comment by roji** [comment]
Duplicate of #23794

**Comment by roji** [comment]
This has already been fixed for the upcoming 5.0.3.
